public/static/js/directives/directives.patch.js

```diff
@@ -8,6 +8,17 @@ directives.patch.directive('patchCommitPanel', function() {
     restrict : 'E',
     templateUrl : '/static/partials/patch_commit_panel.html',
     link : function(scope, element, attrs) {
+      scope.formatByCommit = function(summaries) {
+          var byCommits = {};
+          _.each(summaries, function(summary) {
+              if (byCommits[summary.Description] !== undefined) {
```

Comment:
will the commits be ordered?

Reply:
Some googling: I think we can assume this with reasonable certainty, unless people had commits that were just numbers, but that's unlikely because that's pretty bad.

```
From https://stackoverflow.com/questions/5525795/does-javascript-guarantee-object-property-order:
YES (for non-integer keys).

Most Browsers iterate object properties as:

Integer keys in ascending order (and strings like "1" that parse as ints)
String keys, in insertion order (ES2015 guarantees this and all browsers comply)
Symbol names, in insertion order (ES2015 guarantees this and all browsers comply)```

(Also, the summaries will be in order so insertion order is fine).

Reply:
nice!